pymaker/dss.py

```diff
@@ -238,6 +238,11 @@ def __init__(self, lognote: LogNote):
             self.ilk = str(Web3.toText(lognote.arg1)).replace('\x00', '')
             self.urn = Address(Web3.toHex(lognote.arg2)[26:])
             self.collateral_owner = Address(Web3.toHex(lognote.arg3)[26:])
+            self.dai_recipient = Address(Web3.toHex(lognote.get_bytes_at_index(3))[26:])
+            self.dink = Wad(int.from_bytes(lognote.get_bytes_at_index(4), byteorder="big", signed=True))
```

Comment:
Note we can't use `Web3.toInt` here because it doesn't pass `signed=True` to `int.from_bytes`.